frontend/src/container/Dashboard.jsx

```diff
@@ -481,10 +479,11 @@ class Dashboard extends React.Component {
               <div className='dashboard__workspaceInfo'>
                 <RecentActivity
                   customClass='dashboard__activity'
+                  workspaceId={props.curWs.id}
+                  roleIdForLoggedUser={idRoleUserWorkspace}
```

Comment:
Does this line will print in html the current use role level ? If yes, why use html to store this information, current user informations are probably in javascript data no ?

Reply:
no it wont.
It's to allow the component RecentActivity to know whether it should create a link or not for the folder entries